data/mods/gen3/scripts.ts

```diff
@@ -84,7 +84,7 @@ export const BattleScripts: ModdedBattleScriptsData = {
 
 		if (!this.singleEvent('TryMove', move, null, pokemon, target, move) ||
 			!this.runEvent('TryMove', pokemon, target, move)) {
-			move.mindBlownRecoil = false;
+			if (move.recoil === 'mindblown') move.recoil = undefined; // TODO: ???
```

Comment:
why in Gen 3 concerned with mindblown recoil moves?

Reply:
`git blame`?

Reply:
6c486409f456e7cddc385324bc3cb915fef399c3

Looks like it was copy/pasted from modern code.